README.md

```diff
@@ -1,237 +1,61 @@
-  };
-}
-
-class Car extends Vehicle { // Child class
- constructor(name) {
-    super(name);
-  }
-
-  run() { // Child class method
-    return `Hello! ${super.start()} It has 4 Cylinders.`; // Calling on a method inherited from parent class
-  }
-}
-
-let car1 = new Car("Camry");
-let car2 = new Car("Civic");
-
-// Accessing the child class method which accesses the parent's method
-car1.run();   // "Hello! Engine of Camry starting... It has 4 Cylinders."
-car2.run();   // "Hello! Engine of Civic starting... It has 4 Cylinders."
-```
-
-While on the surface it may look like `Vehicle` and `Car` do the same thing, `class`es that are
-_linked_ and *not* _copied_ are fundamentally a code reuse mechanism--a way for objects to share
-code. `Car` looks to the parent as a reference, and does not have all `Vehicle`'s properties and
-behaviors without it. The way that the code is shared matters. When you attempt to access a
-property or method of an object, JavaScript will searches through the object/`class` and its
-`prototype` until the end of the `prototype` chain is reached. It can create performance and
-code maintenance issues when misapplied.
-
-### Strong Behavioral Subtyping
-
-When creating child `class`es, we defer to [SOLID](https://en.wikipedia.org/wiki/SOLID). SOLID is
-a set of design principles for OO programming that is intended to make software designs more
-understandable, flexible and maintainable. The 'L' in SOLID stands for is Liskov's substitution
-principle:
-
-> objects in a program should be replaceable with instances of their subtypes without altering
-the correctness of that program.
-
-To adhere to good design, we try to maintain _[(strong) behavioral subtyping](https://en.wikipedia.org/wiki/Liskov_substitution_principle)_
-in [programming language theory](https://en.wikipedia.org/wiki/Programming_language_theory).
-A child class instance should be able to replace a parent and behave effectively the same way.
-
-#### Breaks LSP Principle:
-
-```js
-class Reptile {
-  constructor(name) {
-    this.name = name;
-  }
-    crawl(){
-      return `${this.name} crawls away`
-    }
-}
-
-class Lizard extends Reptile {} // Lizard can crawl because it has legs.
-class Snake extends Reptile {} //  Snake cannot crawl because it does not have legs.
-```
-
-Snake class is a subtype of class Reptile, but it can't use the `crawl` method. That means this
-would be breaking the LSP principle.
-
-#### Follows LSP Principle:
-
-```js
-class Reptile {}
-
-class LeggedReptile extends Reptile {
-  constructor(name) {
-    this.name = name;
-  }
-    crawl(){
-      return `${this.name} crawls away`
-    }
-}
+let dog = new Dog("Shadow");
+let cat = new Dog("Missy");
+let bird = new Dog("Tiki");
 
-class Lizard extends LeggedReptile {}
-class Snake extends Reptile {}
+dog.speak(); // Shadow makes a loud sound!
+cat.speak(); // Missy makes a loud sound!
+bird.speak(); // Tiki makes a loud sound!
 ```
 
```

Comment:
This is good. I think though that this lab can include an example of a child with a unique method as well as the inherited `speak`. Just to make it clearer that extension means you can inherit a `clas` and then build additions
in a child

Reply:
By an extra method, I mean something entirely unrelated to the parent (so no need for super).

Like `Bird` can have a `get fly() {}`. Bird can still use `speak` but now has its own functionality

Reply:
gotcha